examples/bluetooth/ble_temperature.py

```diff
@@ -76,7 +83,7 @@ def demo():
     while True:
         # Write every second, notify every 10 seconds.
         i = (i + 1) % 10
-        temp.set_temperature(t, notify=i == 0)
+        temp.set_temperature(t, notify=i == 0, indicate=False)
```

Comment:
was it intended to pass `indicate=True` for some values of `i` to test/demo indications?

Reply:
It wasn't, but happy to add that if you want. I was more just making it clear how someone could modify this if they wanted to use indicate instead.

Reply:
ok, let's leave it as-is for now